Gems/AtomLyIntegration/EMotionFXAtom/Code/Tools/EMStudio/AnimViewportRenderer.cpp

```diff
@@ -251,13 +251,19 @@ namespace EMStudio
         skyBoxFeatureProcessorInterface->SetCubemapRotationMatrix(rotationMatrix);
 
         // Reset ground entity
-        AZ::Transform groundTransform = AZ::Transform::CreateIdentity();
-        AZ::TransformBus::Event(m_groundEntity->GetId(), &AZ::TransformBus::Events::SetLocalTM, groundTransform);
+        AZ::Transform identityTransform = AZ::Transform::CreateIdentity();
+        AZ::TransformBus::Event(m_groundEntity->GetId(), &AZ::TransformBus::Events::SetLocalTM, identityTransform);
```

Comment:
You might just call the UpdateGroundplane() function here after resetting the actor positions below.

Reply:
Change has not been merged yet. But yeah that's a good idea.